### ISSUE TITLE
Negative RegEx-Filters

### ISSUE THREAD

**Issue by eopo** [pagermon/pagermon#201]
Allow Filters to be negated, like:

Condition: ^([0-5]|[7-9])
Negate: true

Would only allow Messages starting with 6 - Well, this example could be solved with ^6, but I guess you know what I mean :)

**Comment by davidmckenzie** [comment]
My bad, it's `?!`: https://www.regextester.com/15

**Comment by eopo** [comment]
Nope. Highlighting/Replacing and Ignoring are two completely separate things, aren’t they?
